src/main/resources/application.yaml

```diff
@@ -49,5 +56,13 @@ oidc:
 ccd:
   data-store:
     host: ${CCD_DATA_STORE_HOST:http://localhost:4452}
+    whitelisted-urls: /ccd/searcCases\?ctid.*
+zuul:
+  routes:
+    search-cases:
```

Comment:
why `search-cases` here?

Reply:
We can pick any name. Will change to 'data-store' that is more appropriate.